src/Illuminate/Database/Eloquent/Factory.php

```diff
@@ -107,6 +99,25 @@ public function createAs($class, $name, array $attributes = [])
     }
 
     /**
+     * Load factories from path.
+     *
+     * @param  string  $path
+     * @return $this
+     */
+    public function load($path)
+    {
+        $factory = $this;
```

Comment:
Because the factories file need it. E.g: <https://github.com/laravel/laravel/blob/master/database/factories/ModelFactory.php>